Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -254,6 +254,28 @@ public void SetLogUrl(string logUrl)
 #endif
     }
 
+    public void ClearLogUrl()
+    {
+        LogUrl.text = "";
+        LogUrlCached = null;
+        PlayerPrefs.DeleteKey(LogUrlKey);
+        AppCenter.SetLogUrl(null);
+#if UNITY_ANDROID && !UNITY_EDITOR
+        AndroidUtility.SetPreferenceString(LogUrlAndroidKey, null);
+#endif
+    }
+
+    public void ClearAppSecret()
+    {
+        AppSecret.text = "";
+        AppSecretCached = null;
+        PlayerPrefs.DeleteKey(AppSecretKey);
+        AppCenter.ParseAndSaveSecretForPlatform(null);
+#if UNITY_ANDROID && !UNITY_EDITOR
+        AndroidUtility.SetPreferenceString(AppSecretAndroidKey, null);
```

Comment:
How does it work on iOS then?

Reply:
Via `PlayerPrefs`. The workaround is needed only for Android.

Reply:
Let's move `PlayerPrefs` stuff to `#if !UNITY_ANDROID` then

Reply:
It works this way. Let's leave a comment in code.

Reply:
@annakocheshkova Can you explain why we need a workaround only for Android?

Reply:
Done